src/amo/components/Search/SearchResult/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+
+import './style.scss';
+
+export default class SearchResult extends React.Component {
+  static propTypes = {
+    result: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { result } = this.props;
+    return (
+      <li className="SearchResult">
+        <Link to={`/en-GB/firefox/addon/${result.slug}/`}
+              className="SearchResult-link"
+              ref={(el) => { this.name = el; }}>
+          <div>
+            <img className="SearchResult-icon" src={result.icon_url}
+                 alt="Icon" />
```

Comment:
This doesn't really add any information and as a result it's probably better being an empty string since all the useful descriptive data is here in other places.